tests/Resources/ti.ui.ios.webviewconfiguration.addontest.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe.ios('Titanium.UI.iOS', function () {
+	
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: tests/Resources/ti.ui.ios.webviewconfiguration.addontest.js;
  Line: 14;
-->
- :no_entry_sign: tests/Resources/ti.ui.ios.webviewconfiguration.addontest.js line 14 – Trailing spaces not allowed. (no-trailing-spaces)